packages/hooks/src/NFT/useNFTViewMethod.ts

```diff
@@ -15,20 +15,31 @@ const fetchNFTView = async (
     methodName: string,
     serializedParams: string
 ) => {
-    const near = await connect({
-        networkId,
-        nodeUrl,
-        deps: {
-            keyStore: undefined,
-        },
-    })
-    const account = await near.account(null)
+    let contract
 
-    const contract = await new Contract(
-        account,
-        contractAddress,
-        getContractMethods('nft')
-    )
+    if (window.near_contracts && window.near_contracts[contractAddress]) {
```

Comment:
I think using `window` is not a good approach, in general. I was thinking more of something coming from React storage functionalities like [unstated-next](https://github.com/jamiebuilds/unstated-next) that is already installed in hooks. What do you think?

Reply:
I could have done that using `useNFTContract` but I wanted to make `useNFTViewMethod` different than `useNFTMethod`, so we don't need to always have a container. 
Just one call to the hook and you get the results you want without adding anything aditional. It will be mainly useful for independent components that are reusable outside cura.run like `NFTE` and maybe cura gallery.

Also, I found that `window` was a popular way to store contract instances, specially in ethereum apps

Reply:
As far as I remember, from doing Ethereum front-end, it was not the best dev experience and some of the choices there are not optimal :)